config/database.yml

```diff
@@ -21,6 +21,8 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: postgres
+  password: io
```

Comment:
If we leave them blank, how do we deal with configuring it locally? 

Should we just use dotenv for this, since the `.env` file isn't checked in? We could just set username/passwords for dev/test in our own local `.env`, right?